### DISCUSSION TITLE
I have a error while starting my bot

### DISCUSSION THREAD

**Discussion by NoPro200** [discord-jda/JDA#2680] [Questions and Help]
[JDA Gateway-Worker 1] WARN net.dv8tion.jda.internal.handle.GuildSetupController - Automatically marking 2 guilds as unavailable due to timeout!

### I can only use my bot when this message comes and that usually takes 1 minute and 19 - 21 seconds

**Comment by codenamehaven** [comment]
Hi @NoPro200,👋

Did you maybe forget to specify intents for your builder? If you're using `new JDABuilder()`, that could be the issue. Try using `JDABuilder.createDefault("YOUR_TOKEN")` which is the default recommended intents.
